apis/v1alpha1/ack-generate-metadata.yaml

```diff
@@ -0,0 +1,17 @@
+ack_generate_info:
+<<<<<<< HEAD
+  build_date: "2023-11-10T17:22:09Z"
+=======
+  build_date: "2023-11-10T17:30:39Z"
+>>>>>>> 18b3556 (Add CertificateSigningRequest Status field and e2e tests)
```

Comment:
Can you please resolve this conflict? regenerating the controller would solve it normally.